pkg/operator/resource/resourceapply/admissions_test.go

```diff
@@ -0,0 +1,213 @@
+package resourceapply
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+
+	admissionv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+	clienttesting "k8s.io/client-go/testing"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+func TestApplyValidatingWebhookConfiguration(t *testing.T) {
+	tests := []struct {
+		name     string
+		existing []runtime.Object
+		input    *admissionv1.ValidatingWebhookConfiguration
+
+		expectedModified bool
+		verifyActions    func(actions []clienttesting.Action, t *testing.T)
+	}{
+		{
+			name: "create",
+			input: &admissionv1.ValidatingWebhookConfiguration{
+				ObjectMeta: metav1.ObjectMeta{Name: "foo"},
+			},
+
+			expectedModified: true,
+			verifyActions: func(actions []clienttesting.Action, t *testing.T) {
+				if len(actions) != 2 {
+					t.Fatal(spew.Sdump(actions))
+				}
+				if !actions[0].Matches("get", "validatingwebhookconfigurations") || actions[0].(clienttesting.GetAction).GetName() != "foo" {
+					t.Error(spew.Sdump(actions))
+				}
+				if !actions[1].Matches("create", "validatingwebhookconfigurations") {
+					t.Error(spew.Sdump(actions))
+				}
+				expected := &admissionv1.ValidatingWebhookConfiguration{
+					ObjectMeta: metav1.ObjectMeta{Name: "foo"},
+				}
+				actual := actions[1].(clienttesting.CreateAction).GetObject().(*admissionv1.ValidatingWebhookConfiguration)
+				if !equality.Semantic.DeepEqual(expected, actual) {
+					fmt.Printf("object, expectd %#v, actual %#v", expected, actual)
```

Comment:
nit: Are you intentionally using`fmt.Printf` instead of `t.Logf`, or was this line an accidental inclusion?

Reply:
unintentional, removed..